.github/workflows/scripts/detect_ts_ignore.py

```
@@ -0,0 +1,37 @@
+import argparse
+import sys
+
+def check_ts_ignore(files):
```

Ruff (0.8.2)

Missing return type annotation for public function `check_ts_ignore`

Add return type annotation: `int`

(ANN201)